src/shared/components/oncoprint/controls/OncoprintControls.tsx

```diff
@@ -1551,12 +1551,11 @@ export default class OncoprintControls extends React.Component<
                     <Button
                         active={this.showMinimap}
                         onClick={this.toggleShowMinimap}
-                        className="oncoprint__controls__minimap_button"
```

Comment:
why remove this?

Reply:
Because the only reference to this class is a removed entry in `src/shared/components/oncoprint/controls/styles.scss`

Reply:
fair enough